mobile/library/cc/engine_builder.h

```diff
@@ -18,8 +19,16 @@
 namespace Envoy {
 namespace Platform {
 
+constexpr int DefaultJwtTokenLifetimeSeconds = 31536000; // 1 year
 constexpr int DefaultXdsTimeout = 5;
 
+// Represents the locality information in the Bootstrap's node.
+struct NodeLocality {
```

Comment:
Nit: Consider moving to its own file.

Reply:
I thought about this, but preferred to keep it in engine_builder.h b/c someone would only use it in the context of setNodeLocality, which is an EngineBuilder function, so they would only have to include one file, and it is small enough of a struct.

Reply:
Yeah it's fine